tsconfig.build.json

```diff
@@ -13,6 +13,7 @@
     }
   },
   "angularCompilerOptions": {
+    "enableIvy": false,
```

Comment:
Is this needed so that our lib works for non-ivy too?

Reply:
Yes, turn enableIvy only need after upgrade to Angular 10
 you will need to enable this and then, the library will be supported only by Angular 9 and Angular 10, and then the NGXS library will be supported only by Angular 9 and Angular 10